state-chain/runtime/src/chainflip/solana_elections.rs

```diff
@@ -652,6 +702,11 @@ impl
 			ingress_identifiers,
 			&block_height,
 		)?;
+		SolanaVaultSwapTracking::on_finalize(
+			&mut vault_swap_translator.translate_electoral_access(generic_electoral_access),
+			vault_swap_identifiers,
+			&crate::System::current_block_number(),
```

Comment:
when rebased, the liveness PR already calls this, so we can pass the same block number in rather than call twice